common/event_emitter.go

```diff
@@ -180,7 +197,7 @@ func (e *BaseEventEmitter) emit(event string, data interface{}) {
 				handlers = append(handlers[:i], handlers[i+1:]...)
 				continue
 			default:
-				go emitEvent(handler)
+				emitEvent(handler)
```

Comment:
Bye bye goroutine -- this helps keep the order of incoming events.

Reply:
😱